SystemTests/roles/robot/files/k8s_methods.robot

```diff
@@ -66,3 +66,13 @@ Submit SparkPi Job
     ${submitbody}=    Create Dictionary   name=${job_name}   language=Scala   main_class=org.apache.spark.examples.SparkPi    path_to_main_app_file=local:///opt/spark/examples/jars/spark-examples_2.11-2.4.0.jar
     ${response}=    Post Request With Json Body   /piezo/submitjob    ${submitbody}
     [return]  ${response}
+
+Wait For Spark Job To Finish
+    [Arguments]    ${job_name}
+    :For    ${i}    IN RANGE   0    24    
+    \   Sleep     5 seconds
+    \   ${response}=   Get Status Of Spark Job   ${job_name}
+    \   ${message}=  Get Response Data Message     ${response}
+    \   ${finished}=    Set Variable If     '${message}'=='COMPLETED'   ${True}     ${False}
+    \   Exit For Loop If    ${finished}==${True}
```

Comment:
Out of curiosity, would `Exit For Loop If    ${finished}` work here?

Reply:
It turns out it does 146be93